src/commands/promote.ts

```diff
@@ -0,0 +1,112 @@
+import { MatrixClient, MessageEvent, MessageEventContent, MembershipEvent } from "matrix-bot-sdk"
+
+import { CommandError, sendMessage, sleep } from "src/utils"
+
+
+export const PROMOTE_COMMAND = "promote"
+export const POWER_LEVEL_ALIAS: Record<string, number> = {
+  default: 0,
+  moderator: 50,
+  admin: 100
+}
+const MATRIX_HOMESERVER = "matrix.parity.io"
```

Comment:
with new server it's still the same? i think it could be moved to separate constants or env